Dockerfile.rhel

```diff
@@ -12,47 +12,45 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM registry.access.redhat.com/rhel7/rhel
+FROM registry.access.redhat.com/ubi8/ubi
 
 MAINTAINER Sonatype <cloud-ops@sonatype.com>
 
 LABEL name="Nexus IQ Server image" \
   vendor=Sonatype \
-  version="1.84.0-01" \
-  release="1.84.0" \
+  version="1.85.0-01" \
+  release="1.85.0-01" \
```

Comment:
should "release" remain the shorter version (1.85) lacking the build number (-01).

Reply:
> should "release" remain the shorter version (1.85) lacking the build number (-01).

👍  Fixed